### PR TITLE
Update file path for Franka Emika Panda

### PR DESCRIPTION
**Pull request by cinephilia-z**
from https://github.com/KavrakiLab/robowflex_resources/pull/7 (Updated panda description #7)

### CODE REVIEW

File: robowflex_library/scripts/resources_visualization.cpp [hunk 1]
```diff
@@ -38,10 +38,10 @@ const std::string ur5[4] = {
 
 // Franka Emika Panda robot with hand
 const std::string panda[4] = {
-    "package://robowflex_resources/panda/urdf/panda_arm_hand.urdf.xacro",    // urdf
-    "package://robowflex_resources/panda/config/panda_arm_hand.srdf.xacro",  // srdf
-    "package://robowflex_resources/panda/config/joint_limits.yaml",          // joint limits
-    "package://robowflex_resources/panda/config/kinematics.yaml"             // kinematics
+    "package://robowflex_resources/panda/urdf/panda_arm_hand.urdf.xacro",     // urdf
```
Comment (ChamzasKonstantinos) [inline robowflex_library/scripts/resources_visualization.cpp:41]: I think the new files are named ``package://robowflex_resources/panda/panda_description/urdf/panda.urdf `` etc.

File: robowflex_library/scripts/resources_visualization.cpp [hunk 2]
```diff
@@ -38,10 +38,10 @@ const std::string ur5[4] = {
...
+    "package://robowflex_resources/panda/config/panda_arm_hand.srdf.xacro",   // srdf
```
Comment (ChamzasKonstantinos) [inline robowflex_library/scripts/resources_visualization.cpp:42]: See above

**Review by ChamzasKonstantinos — CHANGES_REQUESTED** [review]
I think you did not use the new names you have introduced. 
Also to avoid lintint errors, either install clang-format in your IDE or simply run this script at the top level directory : https://github.com/KavrakiLab/robowflex/blob/master/format.sh

File: robowflex_library/scripts/resources_visualization.cpp (hunk 1 above)
Comment (cinephilia-z) [reply robowflex_library/scripts/resources_visualization.cpp:41]: yes. wrong commit. sorry about that.
Thanks for catching it!

File: robowflex_library/scripts/resources_visualization.cpp [hunk 3]
```diff
@@ -38,10 +38,10 @@ const std::string ur5[4] = {
 
 // Franka Emika Panda robot with hand
 const std::string panda[4] = {
-    "package://robowflex_resources/panda/urdf/panda_arm_hand.urdf.xacro",    // urdf
-    "package://robowflex_resources/panda/config/panda_arm_hand.srdf.xacro",  // srdf
-    "package://robowflex_resources/panda/config/joint_limits.yaml",          // joint limits
-    "package://robowflex_resources/panda/config/kinematics.yaml"             // kinematics
+    "package://robowflex_resources/panda/panda_description/urdf/panda.urdf",  // urdf
```
Comment (ChamzasKonstantinos) [inline robowflex_library/scripts/resources_visualization.cpp:41]: This still needs to be updated

**Review by ChamzasKonstantinos — CHANGES_REQUESTED** [review]

**Review by ChamzasKonstantinos — APPROVED** [review]
LGTM!

**Pull request merged**